src/test_request.erl

```diff
@@ -51,6 +51,9 @@ delete(Url) ->
 delete(Url, Opts) ->
     request(delete, Url, [], [], Opts).
```

Comment:
fair point. I don't know, tbh, I don't feel particularly strong about it, but using smaller arity functions as defaults for same functions with bigger arity is pretty standard erlang idiom, so if you up to a bigger change I'll +1 it.